### PR TITLE
[CF-361][FT] Fixed data collector in tests for rollbacks

### PR DESCRIPTION
**Pull request by ntymtsiv**
Added password updating for migrated users on dst cloud.
To get key pairs we have to log in as each user and get list
of key pairs he owns. Since user password can not be migrated,
need to update it before log in.

### CODE REVIEW

**Comment by SVilgelm** [comment]
@plomakin Please take a look at this patch

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: devlab/tests/data_collector.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
 from devlab.tests import base
 import devlab.tests.utils as utils
 
+from keystoneclient import exceptions as ks_exceptions
```
Comment (SVilgelm) [inline devlab/tests/data_collector.py]: move this import into the block with other external libraries (`yaml`)

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module devlab.tests.data_collector
devlab/tests/data_collector.py:101: [W0631(undefined-loop-variable), 
DataCollector.nova_collector.get_user] Using possibly undefined loop variable 'user'
devlab/tests/data_collector.py:104: [W0631(undefined-loop-variable), 
DataCollector.nova_collector.get_user] Using possibly undefined loop variable 'user'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module devlab.tests.data_collector
devlab/tests/data_collector.py:101: [W0631(undefined-loop-variable), 
DataCollector.nova_collector.get_user] Using possibly undefined loop variable 'user'
devlab/tests/data_collector.py:104: [W0631(undefined-loop-variable), 
DataCollector.nova_collector.get_user] Using possibly undefined loop variable 'user'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1

File: devlab/tests/data_collector.py [hunk 2]
```diff
@@ -89,28 +90,51 @@ def nova_collector(self, destination, resources):
...
         if 'keypairs' in resources:
-            def get_user(name):
+            def get_user():
+                def get_tenant_name(t_id):
+                    for t in cloud_tenants:
+                        if t_id == t.id:
+                            return t.name
+                    raise RuntimeError('Tenant with id "%s" not found' % t_id)
+
                 for _user in users:
```
Comment (SVilgelm) [inline devlab/tests/data_collector.py]: remove underscore

**Comment by SVilgelm** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module devlab.tests.data_collector
devlab/tests/data_collector.py:101: [W0631(undefined-loop-variable), 
DataCollector.nova_collector.get_user] Using possibly undefined loop variable 'user'
devlab/tests/data_collector.py:104: [W0631(undefined-loop-variable), 
DataCollector.nova_collector.get_user] Using possibly undefined loop variable 'user'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
Please fix pylint issues

**Comment by SVilgelm** [comment]
@plomakin, I think you can take this patch and fix all problems.

**Comment by plomakin** [comment]
Closing because of #939

**Pull request closed without merging**